docs/development/Setup.md

```diff
@@ -56,11 +62,13 @@ finish before doing anything! Once finished, you should see the workspace below.
 ![Initial workspace](oomph/install-4.png)
 
 
+#### Text file encoding
+
+If your are using Windows, make sure the text file encoding is set to `UTF-8`. This can be done under Windows > Preferences > General > Workspace > Text file encoding. Otherwise you might encounter `Unexpected character` errors.
+
 #### Change the target platform
 
-The target platform selected during the installation can be changed, even if it is a bit hidden. In the main menu, click _Help_ > _Perform Setup tasks_ then click _Back_. Select
-the _Show all variables_ checkbox and change the _Buildship Target Platform_ to the desired version. Click _Next_ and _Finish_, then wait for the target platform to download. Once
-finished, the source code will be compiled against the new target platform.
+The target platform selected during the installation can be changed, even if it is a bit hidden. Open one of the `tooling-eXXX.target` files and select `Set as Active Target Platform`
 
```

Comment:
I think what is yet missing here is the step to set the `jdk8.location` and `jdk11.location` (in `~/.gradle/gradle.properties`) as per @donat's [comment](https://github.com/eclipse/buildship/pull/1154#pullrequestreview-963599095)